### PR TITLE
Extend the readme

### LINKED ISSUE #99: Extend the Readme
Add information about features, and usage

### PR DESCRIPTION
**Pull request by VictorKuenstler**
Closes #99

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,27 +6,96 @@
...
 
+`streams-bootstrap` provides base classes and utility functions for Kafka Streams applications.
+
+It provides a common way to
+- configure Kafka Streams applications
+- deploy streaming applications into a Kafka cluster on Kubernetes via Helm Charts
+- reprocess data
+
+Visit our [blogpost](https://medium.com/bakdata/continuous-nlp-pipelines-with-python-java-and-apache-kafka-f6903e7e429d) and [demo](https://github.com/bakdata/common-kafka-streams-demo) for an overview and a demo application.  
+The common configuration and deployments on Kubernetes are supported by the [Streams-Explorer](https://github.com/bakdata/streams-explorer), which makes it possible to explore and monitor data pipelines in Apache Kafka.
```
Comment (disrupted) [inline README.md:17]: I'd say either "Streams Explorer" or "streams-explorer"

File: README.md [hunk 2]
```diff
@@ -6,27 +6,96 @@
 
 # streams-bootstrap
 
+`streams-bootstrap` provides base classes and utility functions for Kafka Streams applications.
+
+It provides a common way to
+- configure Kafka Streams applications
+- deploy streaming applications into a Kafka cluster on Kubernetes via Helm Charts
```
Comment (philipp94831) [inline README.md:13]: deploy streaming application on Kubernetes via Helm charts

File: README.md [hunk 3]
```diff
@@ -6,27 +6,96 @@
...
+        input.to(this.getOutputTopic());
+    }
+
+    @Override
+    public String getUniqueAppId() {
+        return "streams-bootstrap-app";
+    }
+
+}
+```
```
Comment (philipp94831) [inline README.md:70]: Maybe override kafka properties but comment that it is optional?

**Review by philipp94831 — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (VictorKuenstler) [reply README.md:78]: Like this?

File: README.md [hunk 4]
```diff
@@ -6,27 +6,104 @@
 
 # streams-bootstrap
 
+`streams-bootstrap` provides base classes and utility functions for Kafka Streams applications.
+
+It provides a common way to
+- configure Kafka Streams applications
+- deploy streaming application on Kubernetes via Helm charts
```
Comment (philipp94831) [inline README.md:13]: applications

**Review by philipp94831 — APPROVED** [review]

**Comment by disrupted** [comment]
Could you add "Closes #99" for reference?

**Review by disrupted — APPROVED** [review]

**Pull request merged**